src/Castle.Windsor.Tests/Bugs/IoC-267.cs

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if FEATURE_APPDOMAIN
```

Comment:
This probably should be `FEATURE_CODEDOM` which comes back in .NET Standard 2.0.

The line in the unit test using `AppDomain` could easily be changed to:
```
var coreAssembly = typeof(Castle.Core.ProxyServices).Assembly.Location;
```